src/pages/Register/Register.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import axios from "axios";
+import "./Register.css";
+import { useAuth } from "../../context/auth-context";
+import { Link, useNavigate } from "react-router-dom";
+import RegisterPic from "../../assets/register.svg";
+import Navbar from "../../Components/Navbar/Navbar";
+
+const Register = () => {
+  const { setUser, setToken } = useAuth();
+  const navigate = useNavigate();
+  const registerHandler = async () => {
+    try {
+      const res = await axios.post("/api/auth/signup", {
+        firstName,
+        lastName,
+        email,
+        password,
+      });
+      console.log(res);
+      if (res.status === 201 && res.data.encodedToken) {
+        setToken(res.data);
+        setUser(res.data.foundUser);
+        navigate("/mynotes");
+      }
+    } catch (error) {
+      console.log(error);
+      console.log(error.response);
+    }
+  };
+
+  const [firstName, setFirstName] = useState("John");
+  const [lastName, setLastName] = useState("Doe");
+  const [email, setEmail] = useState("john@gmail.com");
+  const [password, setPassword] = useState("john123");
+
+  return (
+    <>
+      <Navbar />
```

Comment:
I have seen you are including the navbar in all pages. So you can simply move it to your app.js file

Reply:
Great catch @Maianki ! will refactor this in upcoming PR